scripts/js/commands/checkInternalLinks.ts

```diff
@@ -178,42 +187,27 @@ async function determineCurrentDocsFileBatch(
     // Release notes referenced in files.
     "docs/api/qiskit/release-notes/index.mdx",
     "docs/api/qiskit/release-notes/0.45.mdx",
-    "docs/api/qiskit/release-notes/1.1.mdx",
-    "docs/api/qiskit/release-notes/1.2.mdx",
+    "docs/api/qiskit/release-notes/1.*.mdx",
     // Used by release notes.
     "docs/api/qiskit-ibm-runtime/0.27/qiskit_ibm_runtime.options.ResilienceOptions.mdx",
   ];
 
-  if (!args.currentApis) {
-    toCheck.push(`!{public,docs}/api/*/*`);
-    toLoad.push(`docs/api/*/*`);
-  }
-
-  if (args.qiskitReleaseNotes) {
-    const currentVersion = JSON.parse(
+  if (args.currentApis) {
+    const currentQiskitVersion = JSON.parse(
```

Comment:
We now check the release notes file for the current version if `--current-apis` is set. Before, you had to also set `--qiskit-release-notes`